lib/ddtrace/pin.rb

```diff
@@ -18,6 +18,8 @@ def self.get_from(obj)
     attr_accessor :tracer
     attr_accessor :config
 
+    alias service_name= service=
```

Comment:
👍 this makes easy to keep the `Pin` an implementation detail with a configuration like any other integration.